src/templates/_macros/market_button.html

```diff
@@ -9,7 +9,7 @@
         {{ data_attrs|make_data_attrs }}
         {{ 'disabled' if app.disabled }}
         data-product="{{ app|json }}"
-        {% if app.isWebsite %}href="{{ app.url }}"{% endif %}>
+        {% if app.isWebsite %}href="{% if app.mobile_url %}{{ app.mobile_url }}{% else %}{{ app.url }}{% endif %}"{% endif %}>
```

Comment:
For future cases, where we might show websites to desktop, should this be dependent on if the device is a mobile device? I.e. if it's a mobile device and there's a `mobile_url`, use that, otherwise use the regular `url`.